Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOProjectCard.swift

```diff
@@ -5,15 +5,20 @@ import SwiftUI
 
 struct PPOProjectCard: View {
   @StateObject var viewModel: PPOProjectCardViewModel
+  var parentSize: CGSize
+
+  var onShowProject: ((PPOProjectCardModel) -> Void)? = nil
```

Comment:
This should actually be the `viewBackingDetails` navigation action that I just added. But this misunderstanding is why we've proposed that tapping on the project should open the project page instead of the backing details webview in v2.

Reply:
Removed showProject and renamed any existing APIs, "showProject" -> "viewBackingDetails" and "onShowProject" -> "onViewBackingDetails" etc.